src/components/ProjectCardWithPeople/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReactComponent as BrandImage } from 'assets/images/metadata/brand-image.svg';
+import clsx from 'clsx';
+import { OrganisationProjects } from 'interfaces/organisation';
+import { FC } from 'react';
+import { useNavigate } from 'react-router-dom';
+import getRandomString from 'utils/getRandomString';
+import styles from './index.module.scss';
+
+interface IProjectCardWithPeople {
+  flProjects: OrganisationProjects[];
+}
+
+const ProjectCardWithPeople: FC<IProjectCardWithPeople> = ({ flProjects }) => {
+  const navigate = useNavigate();
+
+  return (
+    <div className={styles[`project-card-with-people`]}>
+      {flProjects.map((flProject: OrganisationProjects) => (
```

Comment:
The `.map()` should not happen in a component which is made to render a card. It should be happen in its parent component.